resources/tax_type/de_vat.json

```diff
@@ -19,7 +19,13 @@
                 {
                     "id": "de_vat_standard_2020",
                     "amount": 0.16,
-                    "start_date": "2020-07-01"
+                    "start_date": "2020-07-01",
+                    "end_date": "2021-01-01"
```

Comment:
The end date is supposed to be the last day of previous month, 2020-12-31 in this case.

Reply:
done